package.json

```diff
@@ -57,5 +57,8 @@
     "mocha": "^2.2.5",
     "rimraf": "^2.4.2",
     "uglifyjs": "^2.4.10"
+  },
+  "dependencies": {
+    "isarray": "^1.0.0"
```

Comment:
Do we actually need this? I was used to `Array.isArray(arr)`, but I am not sure if that's supported by all major browsers.